File: LanguAI.Backend/LanguAI.Backend/ViewModels/Friendship/FriendshipViewModel.cs
using LanguAI.Backend.Core.Enums;

namespace LanguAI.Backend.ViewModels.Friendship;

public class FriendshipViewModel
{
    public int Id { get; set; }

    public int RequesterId { get; set; }

    public int RecipientId { get; set; }

    public FriendshipStatusEnum Status { get; set; }

    public bool IsCloseFriendship { get; set; }

    public DateTime Created { get; set; }
}

File: LanguAI.Frontend/src/api/models/friendship-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { FriendshipStatusEnum } from '../models/friendship-status-enum';
export interface FriendshipViewModel {
  created?: string;
  id?: number;
  isCloseFriendship?: boolean;
  recipientId?: number;
  requesterId?: number;
  status?: FriendshipStatusEnum;
}
